### DISCUSSION TITLE
reappearance of issue #3099?

### DISCUSSION THREAD

**Discussion by allenbrunson** [getnikola/nikola#3696] [Q&A] [answered]
hey folks,

i just started using the nikola "check" command to find mistakes in my pages (great feature by the way), and it prints this:

[2023-06-26 19:07:45] WARNING: check: Broken link in output/listings/index.html: .DS_Store.html

this sure looks to me like a resurgence of issue 3099, from way back in 2018, which should be fixed?

https://github.com/getnikola/nikola/issues/3099

thanx
allen

**Comment by Kwpolska** [comment]
You seem to have a `.DS_Store.html` instead of `.DS_Store`. Try removing it and rebuilding.

**Comment by allenbrunson** [comment]
no, that is not correct. just double-checked, it is the presense of a .DS_Store file in my /listings directory that is causing the nikola build process to think it should generate a listing for it.

(i am using nikola version 8.2.4, installed via homebrew)

**Reply by Kwpolska** [reply, marked as answer]
Fixed in https://github.com/getnikola/nikola/commit/69a633fb3b920e6728f2961d4050cd8955641d54 .
